cirq-ft/README.rst

```diff
@@ -0,0 +1,21 @@
+.. image:: https://raw.githubusercontent.com/quantumlib/Cirq/master/docs/images/Cirq_logo_color.png
+  :target: https://github.com/quantumlib/cirq
+  :alt: cirq-ft
+  :width: 500px
+
+Cirq is a Python library for writing, manipulating, and optimizing quantum
+circuits and running them against quantum computers and simulators.
+
+This module is **cirq-ft**, which enhances support for fault-tolerant algorithms
+and provides functions for quantum memory management.
```

Comment:
Cirq's github homepage defines it as

> A python framework for creating, editing, and invoking Noisy Intermediate Scale Quantum (NISQ) circuits.

Wikipedia says that

> Cirq is an open-source framework for noisy intermediate scale quantum (NISQ) computers.

Perhaps the README.md should discuss (and maybe even resolve!) this apparent contradiction.

Reply:
Ack.  This PR is to add a minimum package for transferring code from a development non-public repository.
The README file is not yet ready; I have added a TODO marker it needs to be reviewed.